homeassistant/components/screenlogic/number.py

```diff
@@ -0,0 +1,90 @@
+"""Support for a ScreenLogic number entity."""
+import logging
+
+from screenlogicpy.const import BODY_TYPE, DATA as SL_DATA, EQUIPMENT, SCG
+
+from homeassistant.components.number import NumberEntity
+
+from . import ScreenlogicEntity
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+SUPPORTED_SCG_NUMBERS = (
+    "scg_level1",
+    "scg_level2",
+)
```

Comment:
We are generally moving to using `NumberEntityDescription` in new platforms but it may not make sense here with only 2 of them

Reply:
Examples

https://github.com/home-assistant/core/blob/dev/homeassistant/components/wled/number.py#L38
https://github.com/home-assistant/core/blob/dev/homeassistant/components/tuya/number.py#L23

Reply:
I'll look at rolling this in with the future move of some of the chemistry sensors to number entities.